README.md

```diff
@@ -24,7 +24,10 @@ Usage of aws-rotate-key:
     	The profile to use. (default "default")
   -version
     	Print version number (1.0.3)
-  -y	Automatic "yes" to prompts.
+  -y
```

Comment:
My only comment is that this text is the output from `aws-rotate-key -help`, which now will say:

```
  -d	Delete old key without deactivation.
  -profile string
    	The profile to use. (default "default")
  -version
    	Print version number (1.0.3)
  -y	Automatic "yes" to prompts.
```

It is not obvious that the text is from there. And I don't think we need to keep it exactly the same.